js/main.js

```diff
@@ -126,3 +129,87 @@ function handleLiClick(event) {
   $detailsPage.classList.remove('hidden');
   data.view = 'details-page';
 }
+
+var $btnAdd = document.querySelector('.add-to-database');
+var $divDatabaseCharImg = document.querySelector('.database-char-img');
+var $databasePage = document.querySelector('.container-database-page');
+
+$btnAdd.addEventListener('click', handleAddToDatabase);
+
+function handleAddToDatabase(event) {
+
+  var databaseCharacter = event.target.closest('[data-character-id]');
+
+  for (var i = 0; i < data.entries.length; i++) {
+    if (Number(databaseCharacter.getAttribute('data-character-id')) === data.entries[i].charID) {
+      data.database = data.entries[i];
```

Comment:
Does this only ever store one entry? What if the user tries to save multiple entries, are they just forgotten?